Project.toml

```diff
@@ -42,12 +41,14 @@ Enzyme = "7da242da-08ed-463a-9acd-ee780be4f1d9"
 Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
 MakieCore = "20f20a25-4f0e-4fdf-b5d1-57303727442b"
 Metal = "dde4c033-4e86-420c-a63e-0dd931031962"
+NCDatasets = "85f8d34a-cbdd-5861-8df4-14fed0d494ab"
```

Comment:
You need to add `NCDatasets` to the test targets. You also need to change the test files and the test init. You need to add `using NCDatasets` to the `NetCDFWriter` tests specifically.